### DISCUSSION TITLE
How to choose which collections to process?

### DISCUSSION THREAD

**Discussion by andromidasj** [meisnate12/Plex-Meta-Manager#413] [Q&A]
Sorry for the stupid question:

So, I've gone through getting this set up through docker, and I created my config.yml file and set everything up, and ran the program in the console. It has now been processing through dozens of huge (hundreds of movies to 1000+) IMDb lists of different genres. I obviously don't want all of these showing up and I only want to choose a specific few to process through.

I've scrubbed through the Wiki but for some reason can't figure it out - how do I stop Plex Meta Manager from scanning every huge online lists and instead have it scan the selections that I want?

**Comment by meisnate12** [comment]
in your config file under the library section, there are some lines that start with `- git: ` these lines call metadata files from the [configs repo](https://github.com/meisnate12/Plex-Meta-Manager-Configs) so delete these lines to have PMM not run those files

**Reply by andromidasj** [reply]
Thanks. There are only 4 lines though - MovieCharts, Studios, IMDBGenres, & People. I'm guessing there's no way to pick and choose which ones to update within those lists? I'm sure at that point, it'd be better to create my own filters and lists, is that correct?

**Reply by meisnate12** [reply]
you have to run the whole file its not really a pick and choose if you want to do that just download the file and delete the collections you dont want
